### ISSUE TITLE
Added metadata column to pretrained_models.csv

### ISSUE THREAD

**Issue by fracpete** [waikato-ufdl/ufdl-backend#84]
* allow for `metadata` column in `pretrained_models.csv`
* import metadata column value into backend
* the metadata column will contain JSON

**Comment by csterling** [comment]
Also, change the file format from CSV to JSON for ease of management.

**Comment by fracpete** [comment]
@fracpete to add metadata:
* mmdetection: 
  `{"classes": ["__background__","person","bicycle","car","motorcycle","airplane","bus","train","truck","boat","traffic light","fire hydrant","stop sign","parking meter","bench","bird","cat","dog","horse","sheep","cow","elephant","bear","zebra","giraffe","backpack","umbrella","handbag","tie","suitcase","frisbee","skis","snowboard","sports ball","kite","baseball bat","baseball glove","skateboard","surfboard","tennis racket","bottle","wine glass","cup","fork","knife","spoon","bowl","banana","apple","sandwich","orange","broccoli","carrot","hot dog","pizza","donut","cake","chair","couch","potted plant","bed","dining table","toilet","tv","laptop","mouse","remote","keyboard","cell phone","microwave","oven","toaster","sink","refrigerator","book","clock","vase","scissors","teddy bear","hair drier","toothbrush"]}`
* [tensorflow coco labels](https://github.com/tensorflow/models/blob/master/research/object_detection/data/mscoco_label_map.pbtxt):
  `{"classes": ["person","bicycle","car","motorcycle","airplane","bus","train","truck","boat","traffic light","fire hydrant","stop sign","parking meter","bench","bird","cat","dog","horse","sheep","cow","elephant","bear","zebra","giraffe","backpack","umbrella","handbag","tie","suitcase","frisbee","skis","snowboard","sports ball","kite","baseball bat","baseball glove","skateboard","surfboard","tennis racket","bottle","wine glass","cup","fork","knife","spoon","bowl","banana","apple","sandwich","orange","broccoli","carrot","hot dog","pizza","donut","cake","chair","couch","potted plant","bed","dining table","toilet","tv","laptop","mouse","remote","keyboard","cell phone","microwave","oven","toaster","sink","refrigerator","book","clock","vase","scissors","teddy bear","hair drier","toothbrush"]}`

**Comment by fracpete** [comment]
Converted `pretrained_models.csv` to `pretrained_models.json`, e.g.:
```
ufdl-object-detection-app/src/ufdl/object_detection_app/migrations/pretrained_models/pretrained_models.json
```
Uses the `name` of the model as key in the outermost JSON layer.

**Comment by csterling** [comment]
Implemented in:
- b12faa43072f26ee9ccf505bdbd4d9b07a15d0fc
- [ufdl-json-messages](https://github.com/waikato-ufdl/ufdl-json-messages/commit/176063af0f162d1f47cd7f9671e48b837af6ca51)

I've also split the JSON into one-model-per-file for easier maintenance.
